app/uk/gov/hmrc/hecstubs/models/accountOverviewDetails/ReturnStatus.scala

```diff
@@ -24,10 +24,12 @@ sealed trait ReturnStatus extends Product with Serializable {
 
 object ReturnStatus {
 
-  case object ReturnFound extends ReturnStatus { override def value = "Return Found" }
-  case object NoticeToFileIssued extends ReturnStatus { override def value = "Notice to File Issued" }
-  case object NoReturnFound extends ReturnStatus { override def value = "No Return Found" }
-  case object NoAccountingPeriodFound extends ReturnStatus { override def value = "No Accounting Period Found" }
+  case object ReturnFound extends ReturnStatus { override def value = "1" }
+
+  case object NoticeToFileIssued extends ReturnStatus { override def value = "2" }
+
+  case object NoReturnFound extends ReturnStatus { override def value = "3" }
+
```

Comment:
do we not need `NoAccountingPeriodFound `?

Reply:
not anymore - that information is now returned as status `2` under the outer status in the CT response